certora/conf/SocialRecoveryModule.conf

```diff
@@ -17,4 +17,5 @@
     ],
     "loop_iter": "3",
     "optimistic_loop": true,
+    "prover_args": ["-smt_groundQuantifiers false"],
```

Comment:
Can we document in the PR description what this does and why this is needed?

Reply:
Done 👍🏾